app/Models/AttributeTypes.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use App\Models\AttributeValues;
+
+class AttributeTypes extends Model
+{
+    use HasFactory;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'name', 
+    ];
+
+    /**
+     * Get all of the attributesType for the AttributeTypes
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function attributes()
```

Comment:
2 function can't have same name. When you try to access relationship via this function, laravel can't detect what relationship do you want to access.

Reply:
But in models of blogs and categories, I still can use their functions when both functions' name is categories.

Reply:
I have tried to change their names but it still doesn't work